tasks/test_cleanup_qnetd.yml

```diff
@@ -11,8 +11,10 @@
     name:
       - corosync-qnetd
     state: absent
+  when: ansible_facts.pkg_mgr | d() != "ansible.posix.rhel_rpm_ostree"
 
 - name: Make sure qnetd config files are not present
   file:
     path: /etc/corosync/qnetd
     state: absent
+  when: ansible_facts.pkg_mgr | d() != "ansible.posix.rhel_rpm_ostree"
```

Comment:
@tomjelinek this was one of the things I had to add to make `ostree` happy - apparently when it is added back it loses the ownership

Reply:
@richm Oh, I see. I created a ticket https://issues.redhat.com/browse/RHEL-16343